### PR TITLE
Save parent payment so log_entries receives :create! in spec ENG-1365

### PR DESCRIPTION
**Pull request by allisonlarson**

### CODE REVIEW

File: core/spec/models/spree/payment_spec.rb [hunk 1]
```diff
@@ -151,6 +151,7 @@
       end
 
       it "should log the response" do
+        payment.save
```
Comment (jordan-brough) [inline core/spec/models/spree/payment_spec.rb]: you probably want to use `save!` here so you find out if it fails for whatever reason

**Comment by jordan-brough** [comment]
left one minor comment.  this is awesome, thanks for fixing it!

**Comment by jordan-brough** [comment]
you'll submit upstream to spree/spree/2-2-stable also yeah?

File: core/spec/models/spree/payment_spec.rb (hunk 1 above)
Comment (allisonlarson) [inline core/spec/models/spree/payment_spec.rb]: changed! Thanks

**Comment by allisonlarson** [comment]
No problem, and yep, I'm doing that now.

**Comment by jordan-brough** [comment]
awesome :+1:

**Pull request merged**